test/CMakeLists.txt

```diff
@@ -144,6 +144,7 @@ macro(onedpl_add_test test_source_file switch_off_checked_iterators)
 endmacro()
 
 set(_regexp_dpcpp_backend_required "(xpu_api/algorithms|xpu_api/functional|xpu_api/numerics)")
+set(_regexp_switch_off_checked_it  "(xpu_api/algorithms|xpu_api/functional|xpu_api/numerics|test/general/header_order_ranges|test/parallel_api/algorithm/alg.sorting/alg.min.max|test/parallel_api/ranges|test/xpu_api/random/device_tests|test/xpu_api/random/interface_tests|test/xpu_api/random/statistics_tests)")
```

Comment:
I don't think you need to include `xpu_api/algorithms|xpu_api/functional|xpu_api/numerics` in the regex, since all of those tests are covered by the statement `if (_file MATCHES "${_regexp_dpcpp_backend_required}")` and so they will never reach the `else` to be compared against this regex.

Reply:
Good point. Fixed.